lib/id3c/cli/command/etl/fhir.py

```diff
@@ -180,7 +180,13 @@ def process_diagnostic_report_bundle_entry(db: DatabaseSession, bundle: Bundle,
     LOG.debug(f"Processing DiagnosticReport Resource «{entry.fullUrl}».")
 
     for reference in resource.specimen:
-        if not matching_system(reference.identifier, INTERNAL_SYSTEM):
+        if not reference.identifier:
+            specimen = reference.resolved(Specimen)
+            reference.identifier = specimen.identifier[0]
+            if not matching_system(reference.identifier, f"{INTERNAL_SYSTEM}/sample"):
+                continue
+
+        elif not matching_system(reference.identifier, INTERNAL_SYSTEM):
```

Comment:
Why are we checking for two different internal systems endpoints here?

Reply:
Our internal bundles from redcap-det ETLs give use `https://seattleflu.org/sample` as the system for specimens while the Ellume team's bundles just use `https://seattleflu.org`

This was just a quick fix, but we should probably reconcile the two in the future.